internal/handlers/tigris/tigrisdb/insert.go

```diff
@@ -81,11 +68,25 @@ func (tdb *TigrisDB) InsertManyDocuments(ctx context.Context, db, collection str
 		insertDocs[i] = b
 	}
 
-	if _, err := tdb.Driver.UseDatabase(db).Insert(ctx, collection, insertDocs); err != nil {
+	if _, err := tdb.Driver.UseDatabase(db).Insert(ctx, collection, insertDocs); err == nil ||
+		(!IsNotFound(err) && !IsInvalidArgument(err)) {
```

Comment:
`IsInvalidArgument` can be a user facing error as well, anything specific that you're ignoring here?

Reply:
Okay, makes sense.